.buildkite/pipeline.yml

```diff
@@ -26,13 +64,34 @@ steps:
     agents:
       - queue=elastic
 
-  - name: ':skull: Mutest'
-    command: build/run 2.3.3 'rake metrics:mutest'
-    timeout_in_minutes: 10
+  - name: ':rubocop: Rubocop'
```

Comment:
and no I think removing frozen string literals until it works everywhere is preferable